dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -21,6 +21,7 @@
 #include "NotificationsLongRunningProcess_h.h"
 #include <TerminalVelocityFeatures-PushNotifications.h>
 #include "PushNotificationUtility.h"
+#include "ToastNotificationUtility.h"
```

Comment:
What are your plans on adding unit tests for these new APIs?

Reply:
Going to include unit tests for register/unregistering the toast from the LRP. I have plans for another PR that contains unit testing for the rest of the LRP functions :)